src/main/java/org/broadinstitute/consent/http/resources/StudyResource.java

```diff
@@ -159,9 +159,11 @@ public Response deleteStudyById(@Auth AuthUser authUser, @PathParam("studyId") I
       // Remove from ES index
       if (studyDatasetIds != null) {
         studyDatasetIds.forEach(id -> {
-          try {
-            elasticSearchService.deleteIndex(id);
-          } catch (IOException e) {
+          try (Response indexResponse = elasticSearchService.deleteIndex(id, user)) {
```

Comment:
It looks like this will end up trying to update `dataset.indexed_date` on the dataset, but it was already deleted in `datasetService.deleteStudy()` above. I think there's only one case where the index needs to be set to `null`, when called from `DatasetResource.deleteDatasetIndex()`. The other two cases are on study or dataset deletion, in which case the dataset and study records will be deleted.